lib/pavilion/commands/run.py

```diff
@@ -107,6 +107,11 @@ def _generic_arguments(parser):
                  "should be necessary only if the system or user environment "
                  "under which Pavilion runs has changed."
         )
+        parser.add_argument(
+            '--no-purge', action='store_false', default=True, dest='purge',
+            help="Do not perform a module purge before running tests. Modules "
+                 "are purged by default."
+        )
```

Comment:
I would expect this option as a test config option, not as a command line argument. I'd expect it to be set at the host file level.